scripts/container-only/wkdev-setup-devhelp

```diff
@@ -65,18 +65,20 @@ run() {
 
     mkdir -p "${books_dir}"
 
-    echo "Linking documentation:"
-    printf "  Source:\t %s\n" "${docs_dir}"
-    printf "  Destination:\t %s\n" "${books_dir}"
-    printf "  Books:\t"
-    for dir in "${docs_dir}"/*; do
-        echo -n " $(basename "${dir}")"
-        ln --symbolic --force "${dir}" "${books_dir}/$(basename "${dir}")"
-    done
-    echo ""
-
-    echo ""
-    echo "Simply run 'devhelp' to access the documentation."
+    _log_ "$(
```

Comment:
That seems like the only non-mechanical change? Did you check the script still works? Code wise it looks fine.

Reply:
I wasn't even aware of this command existing before, but I tested it now and it seems to work.